views/EditConfig.react.js

```diff
@@ -5,7 +5,7 @@ import 'whatwg-fetch';
 import update from 'immutability-helper';
 import * as _ from 'lodash';
 import colorbrewer from 'colorbrewer';
-import math from 'mathjs';
+import { bignumber, number, add, subtract, divide, abs } from 'mathjs';
```

Comment:
In short--any reason we are updating `mathjs`?

Reply:
Security vulnerability, I was alerted by `npm audit`. https://www.cvedetails.com/cve/CVE-2020-7743/

Reply:
Right, I noticed your PR log after I have commented. Sorry!